main/gnome-remote-desktop/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "gnome-remote-desktop"
+pkgver = "47.1"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dvnc=false", "-Dsystemd=false", "-Dtests=false"]
+hostmakedepends = ["cmake", "meson", "pkgconf", "asciidoc"]
+makedepends = [
+    "cairo-devel",
+    "dbus-devel",
+    "fdk-aac-devel",
+    "ffmpeg-devel",
+    "ffnvcodec-headers",
+    "freerdp3-devel",
+    "fuse-devel",
+    "libdrm-devel",
+    "libei-devel",
+    "libepoxy-devel",
+    "libnotify-devel",
+    "libsecret-devel",
+    "libxkbcommon-devel",
+    "opus-devel",
+    "pipewire-devel",
+    "polkit-devel",
+    "tpm2-tss-devel",
+]
+depends = []
+checkdepends = ["dbus", "pipewire", "mutter", "libgudev-devel"]
+pkgdesc = "GNOME Remote Desktop server"
+maintainer = "Denis Strizhkin <strdenis02@gmail.com>"
+license = "GPL-2.0-or-later"
+url = "https://gitlab.gnome.org/GNOME/gnome-remote-desktop"
+source = f"{url}/-/archive/{pkgver}/{pkgname}-{pkgver}.tar.gz"
+sha256 = "bfdfc646947aaac02e7c3230386a34872f501d6ddd66484154536949f51bf1f1"
+# needs dbus
```

Comment:
and it has it via using dbus-run-session itself, so this comment seems wrong

Reply:
checks don't run https://github.com/chimera-linux/cports/actions/runs/11884781818/job/33113464596?pr=3188#step:8:546

Reply:
i have no idea what that links to but 'no module dbus' is python-dbus

Reply:
looking at the errors the correct comment would be `# needs real /sys /proc for some udev things to work`

Reply:
Yes, seems there is no way to run the tests under bwrap even after I fixed python-dbus and stuff.